.github/workflows/pr-custom-review.yml

```diff
@@ -0,0 +1,27 @@
+name: Check reviews
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - review_requested
+      - review_request_removed
+  pull_request_review:
+
+jobs:
+  pr-custom-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pr-custom-review
+        uses: paritytech/pr-custom-review@v1
+        with:
+          token: ${{ secrets.PRCR_TOKEN }}
+          # 🔒 PROTECTED: Changes to locks-review-team should be approved by the current locks-review-team
+          locks-review-team: locks-review
+          team-leads-team: polkadotleads
```

Comment:
What is that doing?

Reply:
Team leads are also required to approve 🔒 changes. Extracted as a requirement from https://github.com/paritytech/ci_cd/issues/233#issue-1026394650 and documented in https://github.com/paritytech/pr-custom-review#built-in-checks and .

Reply:
Okay, then this should be changed to the same team as I had mentioned above.

Reply:
Solved as per https://github.com/paritytech/substrate/pull/11169